file_formats/cmx_format.mli

```diff
@@ -33,7 +33,8 @@ open Misc
 
 type export_info =
   | Clambda of Clambda.value_approximation
-  | Flambda of Export_info.t
+  | Flambda1 of Export_info.t
```

Comment:
Is there a particular reason not to keep it as Flambda?

Reply:
Just to try to avoid confusion, since the type is being changed anyway.